build.gradle

```diff
@@ -628,6 +633,10 @@ createApp.dependsOn installDist
 //tasks.getByName('githubRelease').dependsOn compute_sha_sums
 
 
+poeditorPull.onlyIf {
+	project.hasProperty('apiKey') && project.hasProperty('projectId')
+}
```

Comment:
it should be replaced with
`poeditorPull.onlyIf {project.hasProperty('POEDITOR_APIKEY')}`

Reply:
@akasolace just tried again to be sure, but as expected it didn't work (it's not enough as a condition).
I can add it, but I cannot remove the other two (that exists only if previous tasks on POEditor succeded).

Reply:
@pierpaolocira  for me the task PoEditor does not run (i.e. it is skipped) if I let  this condition `project.hasProperty('apiKey') && project.hasProperty('projectId')`. 
Maybe what we need is a way to check that `POEDITOR_APIKEY != ""`

Reply:
> @pierpaolocira for me the task PoEditor does not run (i.e. it is skipped) if I let this condition `project.hasProperty('apiKey') && project.hasProperty('projectId')`

Oh, this was not clear to me, sorry... I hope to go for a PR by today, or tomorrow at most.

Reply:
@akasolace I did't understand it was skipping the task for you. Now I tried both with empty value (skipped) and non-empty value (not skipping, but failing on my case because the Key is no valid, as expected).
Please check and let me know

Reply:
@pierpaolocira sounds promising, I will test it tonight ...

Reply:
Great... I hope it works 100% to you, so I can finally merge it to my branch on #372 and start testing that...